ci/restore_questionnaire_state.yaml

```diff
@@ -17,6 +17,12 @@ run:
   args:
     - -exc
     - |
+      if [ -z "${BACKUP_NAME}" ]
+      then
+        echo "BACKUP_NAME variable must be set"
+        exit 1
+      fi
```

Comment:
We should check all the variables. It would still error if other params were missing but BACKUP name was passed?.

Reply:
The other vars have default values so it wouldn't error, if used in pipeline, I mean.

Reply:
It's a loop now and checks all the vars (if they are set), thanks.